Macros/e3 Includes/e3_Basics.inc

```diff
@@ -2418,3 +2419,42 @@ SUB check_Forage
 	}
 /if (${Debug}) /echo <== check_Forage -|
 /RETURN
+
+|----------------------------------------------------------------------------|
+|- Sends instance invites to all those in your group or raid.
+|- Install Instructions:
+|-     Add this SUB to the bottom of e3 Includes/e3_Basics.inc
+|-     Add /doevents sendInstanceInvites to SUB basics_Background_Events
+|- Usage Instructions:
+|-     Use "/bc send instance invites" on the toon in the instance
+|-     to send out instance invites.
+|----------------------------------------------------------------------------|
+#event sendInstanceInvites "<#1#> send instance invites"
```

Comment:
How about "/bc inst invite" so it matches what you put in the chat box? 

Can just add a second event line:
#event sendInstanceInvites "<#1#> inst invite"